addons/web/static/src/core/errors/error_service.js

```diff
@@ -110,6 +110,10 @@ export const errorService = {
 
         browser.addEventListener("unhandledrejection", async (ev) => {
             const error = ev.reason;
+            if (typeof error !== "string" && !(error instanceof Error)) {
```

Comment:
why not string?

Reply:
We have a test that checks it handles strings. IMO this is not correct, only `Error` objects should be handled here.

Reply:
I agree, which test?

Reply:
"can handle rejected promise errors with a string as reason" in `error_service_tests.js:66`

Reply:
Filtering out the string case here is only relevant to make the test pass, but according to the [description of the commit adding the test](https://github.com/odoo/odoo/commit/5d45465758ae46e1838a97ca5c7aeee47d9256c0), simply ignoring rejection of strings would do the trick as well to fix the issue, so I think I would remove it.

Would be nice to have @sdegueldre input on this PR anyway.

Reply:
I don't think we should be ignoring things that are thrown even if they're not errors. This just silently eats real crashes.